### ISSUE TITLE
[IMO] create develop and feature branches

### ISSUE THREAD

**Issue by pureugong** [Kitchu0401/lazecrew-batch-service#8]
It seems like you are working on directly master branch. Why don't you have dev and feature branches. 

http://nvie.com/posts/a-successful-git-branching-model/

**Comment by Kitchu0401** [comment]
Agreed, and I think it's good for me practicing using none-master-branches even though I couldn't expect any co-operation. It's just because of my laziness. 😂

Gonna check article anyway. Good reading stuffs always welcome! :man_dancing:

**Comment by pureugong** [comment]
@Kitchu0401 Please make feature branch even though you are working alone.
It would be quite hard to follow legacy history soon.

**Comment by Kitchu0401** [comment]
@pureugong 
Will take it serisously. Promise! :laughing:

**Comment by Kitchu0401** [comment]
@pureugong 
Created `dev` branch! 🎉
It's gonna be **REAL** git from now on!
